chemie/chemie/models.py

```diff
@@ -13,7 +13,7 @@ class Sponsor(models.Model):
     start_date = models.DateTimeField(auto_now=False, auto_now_add=True)
     end_date = models.DateTimeField(auto_now=False, auto_now_add=False)
     image = ImageField(upload_to='sponsors', verbose_name="Bilde")
-    author = models.ForeignKey(User)
+    author = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
Vil ikke dette kaskade-slette brukeren som opprettet Sponsor-objektet når Sponsor-objektet slettes?

Reply:
Motsatt, dersom du sletta en bruker så sletta du Sponsorobjektet. 
on_delete er by default CASCADE for Django < 2.0

Reply:
Sweet. Gjetter på at dere har testa alt annet slik at det funker?